### PR TITLE
Switch to HTTPS URLs (addition to #8)

### PR DESCRIPTION
**Pull request by Eagle3386**

### CODE REVIEW

File: release/background.js [hunk 1]
```diff
@@ -212,7 +212,7 @@ function updateContextMenus()
     //     WE NEED TO CALL THIS UPON PREFS CHANGES FOR THE TIME BEING TO ENSURE IT'S CREATED BEFORE IT'S NEEDED.
     //     Rejig whenever the above bug is included (allows dynamic menu updating) - not in FF dev as of end Oct '17!!
 
-    /// let isUrl = (activeSelection.search(/^(http|ftp)/) >= 0);
```
Comment (fnxweb) [inline release/background.js:215]: Ew, I need to figure out if that should include **file:** and **javascript:** before they get their act together and fix dynamic menus ...

Comment (Eagle3386) [reply release/background.js:215]: Were you able to progress on this matter?

Comment (fnxweb) [reply release/background.js:215]: Soz, not yet. Very little spare time ATM, used what I had recently to sort out Thunderbird version compatibility.

Comment (Eagle3386) [reply release/background.js:215]: It's all good - just wanted to get a more recent status, so I don't accidentally delete my PR branch.. :wink:

**Pull request merged**